test/lib/tasks/minify.js

```diff
@@ -1,263 +1,164 @@
-		`"mappings":"AACA,SAASA,KAAKC,GACb,IAAIC,EAAYD,EAChBE,QAAQC,IAAIF,EACb,CACAF"}`;
-
-	const resSourceMap = await writer.byPath("/test.js.map");
-	if (!resSourceMap) {
-		t.fail("Could not find /test-dbg.js.map in target locator");
-	}
-	t.deepEqual(await resSourceMap.getString(), expectedSourceMap, "Correct source map content");
 });
 
-test("integration: minify omitSourceMapResources=false (without taskUtil)", async (t) => {
-	const {reader, writer, workspace} = createWorkspace();
-	const content = `
-function test(paramA) {
-	var variableA = paramA;
-	console.log(variableA);
-}
-test();`;
-	const testResource = resourceFactory.createResource({
-		path: "/test.js",
-		string: content
-	});
-	await reader.write(testResource);
-
+test("minify: omitSourceMapResources: true, useInputSourceMaps: false", async (t) => {
+	const {minify, workspace, taskUtil, minifierStub} = t.context;
+	minifierStub.resolves([{
+		resource: "resource A",
+		dbgResource: "dbgResource A",
+		sourceMapResource: "sourceMapResource A",
+		dbgSourceMapResource: "dbgSourceMapResource A" // optional
+	}, {
+		resource: "resource B",
+		dbgResource: "dbgResource B",
+		sourceMapResource: "sourceMapResource B",
+	}]);
 	await minify({
 		workspace,
+		taskUtil,
 		options: {
-			pattern: "/test.js",
-			omitSourceMapResources: false
+			pattern: "**",
+			omitSourceMapResources: true,
+			useInputSourceMaps: false
 		}
 	});
 
-	const expected = `function test(t){var o=t;console.log(o)}test();
-${SOURCE_MAPPING_URL}=test.js.map`;
-	const res = await writer.byPath("/test.js");
-	if (!res) {
-		t.fail("Could not find /test.js in target locator");
-	}
-	t.deepEqual(await res.getString(), expected, "Correct file content");
-
-	const resDbg = await writer.byPath("/test-dbg.js");
-	if (!resDbg) {
-		t.fail("Could not find /test-dbg.js in target locator");
-	}
-	t.deepEqual(await resDbg.getString(), content, "Correct debug-file content");
-
-	const expectedSourceMap =
-		`{"version":3,"file":"test.js",` +
-		`"names":["test","paramA","variableA","console","log"],"sources":["test-dbg.js"],` +
-		`"mappings":"AACA,SAASA,KAAKC,GACb,IAAIC,EAAYD,EAChBE,QAAQC,IAAIF,EACb,CACAF"}`;
-
-	const resSourceMap = await writer.byPath("/test.js.map");
-	if (!resSourceMap) {
-		t.fail("Could not find /test-dbg.js.map in target locator");
-	}
-	t.deepEqual(await resSourceMap.getString(), expectedSourceMap, "Correct source map content");
+	t.is(minifierStub.callCount, 1, "minifier got called once");
+	const minifierCallArgs = minifierStub.firstCall.firstArg;
+	t.deepEqual(minifierCallArgs.resources, ["resource A", "resource B"], "Correct resources provided to processor");
+	t.is(minifierCallArgs.fs, "fs interface", "Correct fs interface provided to processor");
+	t.is(minifierCallArgs.taskUtil, taskUtil, "Correct taskUtil provided to processor");
+	t.deepEqual(minifierCallArgs.options, {
+		addSourceMappingUrl: false,
+		readSourceMappingUrl: false,
+		useWorkers: true
```

Comment:
Maybe add a test case where `useWorkers` is false? (i.e. no taskUtil parameter)

Reply:
Done 👍